File: js_question.js
console.log('Start');
setTimeout(() => {
  console.log('Timeout');
}, 0);
Promise.resolve().then(() => {
  console.log('Promise');
});

(async () => {
  console.log('Async Start');
  await null;
  console.log('Async End');
})();

console.log('End');

// Second heighest salary in employee table

File: Java_Question/JS_Question.java
package Java_Question;

public class JS_Question {

	public static void main(String[] args) {
		String question_1 = """
				console.log('Start'); setTimeout(() => { console.log('Timeout'); }, 0);
				Promise.resolve().then(() => { console.log('Promise'); });
				(async () => { console.log('Async Start'); await null; console.log('Async
				End'); })();
				console.log('End');
					""";

	}
}